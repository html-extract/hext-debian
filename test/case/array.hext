<div>
  <h2 @text:heading/>
  <p @text:paragraph/>
</div>
