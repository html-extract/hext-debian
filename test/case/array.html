<html>
<body>
  <div>
    <h2>Heading1</h2>
    <p>Text1</p>
    <p>Timestamp1</p>
    <h2>Heading2</h2>
    <p>Text2</p>
    <p>Timestamp2</p>
    <div class="advertising">...</div>
    <h2>Heading3</h2>
    <p>Text3</p>
    <p>Timestamp3</p>
  </div>
</body>
</html>
