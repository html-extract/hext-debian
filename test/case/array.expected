{"heading":["Heading1","Heading2","Heading3"],"paragraph":["Text1","Text2","Text3"]}
